app/assets/javascripts/app/discussions/templates/desktop.html.slim

```diff
@@ -22,4 +23,4 @@ TopBarLayout
         label Tags
         input ng-model="ctrl.discussion.tags" type="text"
 
-      textarea.Editor placeholder="Write the fundaments of the discussion" ng-model="ctrl.discussion.body" required=""
+      textarea.Editor.Editor-noSidePadding placeholder="Write the fundaments of the discussion" ng-model="ctrl.discussion.body" required=""
```

Comment:
`noSidePadding` isn't a good name, and please read [this](https://github.com/suitcss/suit/blob/master/doc/README.md), you have been using it for months and still have no idea what you're doing